comp/core/workloadmeta/collectors/internal/remote/processcollector/process_collector.go

```diff
@@ -147,7 +148,10 @@ func (s *streamHandler) IsEnabled() bool {
 	if flavor.GetFlavor() != flavor.DefaultAgent {
 		return false
 	}
-	return s.Reader.GetBool("language_detection.enabled")
+
+	localCollectorEnabled := util.LocalProcessCollectorIsEnabled()
+
+	return s.Reader.GetBool("language_detection.enabled") && !localCollectorEnabled
```

Comment:
small nit:

no need to define `localCollectorEnabled` variable. 

```
return s.Reader.GetBool("language_detection.enabled") && !util.LocalProcessCollectorIsEnabled()
```

Reply:
Fixed